addons/account/models/account_journal_dashboard.py

```diff
@@ -308,6 +310,22 @@ def action_open_reconcile(self):
                 'tag': 'manual_reconciliation_view',
                 'context': action_context,
             }
+    
+    @api.multi
+    def action_open_to_check(self):
+        ids = self.to_check_ids()
+        action_context = {'show_mode_selector': False, 'company_ids': self.mapped('company_id').ids}
+        action_context.update({'mode': 'to_check'})
+        action_context.update({'statement_line_ids': ids})
+        return {
+            'type': 'ir.actions.client',
+            'tag': 'bank_statement_reconciliation_view',
+            'context': action_context,
+        }
+    
+    def to_check_ids(self):
+        statement_line_ids = self.env['account.move'].search([('journal_id', 'in', self.ids), ('to_check', '=', True)]).mapped('line_ids.statement_line_id')
```

Comment:
domain here and on the invoice aren't the same but definitively should (full_reconcile_id clause is missing)